sunyar-fabric.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Sunyar fabric
+parent: Labs
+---
+# Lab Name
+Sunyar fabric
+
+# Short Description
+Sunyar is a settlement distributed system based on Hyperledger fabric. This is a non-profit project for contributing charities to help needy people in a decentralized ecosystem.
```

Comment:
```suggestion
Sunyar is a settlement distributed system based on Hyperledger Fabric. This is a non-profit project for contributing charities to help needy people in a decentralized ecosystem.
```

Reply:
Please note that there is another lab focused on charity tracking and Hyperledger Fabric -- [Karma Charity Platform](https://labs.hyperledger.org/labs/karma-charity-platform.html). This might be a good lab to collaborate with for ongoing work.

Reply:
Sounds great. we are trying to read their project and contribute on required situations.